apis/config/v1beta1/configuration_types.go

```diff
@@ -192,8 +192,22 @@ type WaitForPodsReady struct {
 	// until the jobs reach the PodsReady=true condition. It defaults to false if Enable is false
 	// and defaults to true otherwise.
 	BlockAdmission *bool `json:"blockAdmission,omitempty"`
+
+	// RequeuingTimestamp defines the timestamp used for requeuing a Workload
+	// that was evicted due to Pod readiness. Defaults to Eviction (back of queue).
```

Comment:
I'm wondering if "(back of queue)" could be misleading, there could be lower priority workloads that are even more behind in the queue order, because priority is checked first: https://github.com/kubernetes-sigs/kueue/blob/27902866b88d9f90fcb0a6a296946c12bcadcc6d/pkg/queue/cluster_queue_strict_fifo.go#L50-L55. I would suggest to just remove the text in the brackets.

Reply:
Agree, done